oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexNameHelper.java

```diff
@@ -51,7 +51,8 @@ public static String getIndexAlias(String indexPrefix, String indexPath) {
             throw new IllegalArgumentException("Not an index definition node state");
         }
         PropertyState type = indexNode.getProperty(IndexConstants.TYPE_PROPERTY_NAME);
-        if (type == null || !ElasticIndexDefinition.TYPE_ELASTICSEARCH.equals(type.getValue(Type.STRING))) {
+        String typeValue = type != null ? type.getValue(Type.STRING) : "";
+        if (!ElasticIndexDefinition.TYPE_ELASTICSEARCH.equals(typeValue) && !"disabled".equals(typeValue)) {
```

Comment:
Same as above... that means indexes of type "disabled" are threaded as Elastic indexes, which is wrong.